lib/store/useTemplateStore.tsx

```diff
@@ -363,17 +363,18 @@ const createTemplateStore = (initProps?: Partial<InitialTemplateStoreProps>) =>
               const elIndex = get().form.elements.findIndex((el) => el.id === elId);
               return get().form.elements[elIndex]?.properties.choices?.[choiceIndex];
             },
-            duplicateElement: (itemId, groupId = "") => {
+            duplicateElement: (itemId, groupId = "", copyEn = "", copyFr = "") => {
```

Comment:
Wondering if we should pass the full title in...

The element that's calling duplicate already knows it's own title so we can clear out the work from the template store. 



duplicateElement: (itemId, groupId = "", titleEn = "", titleFr = "") => {

Reply:
In the template store the element doesn't have the latest id/val which makes it difficult to do the lookup for the element there.